### ISSUE TITLE
🐛 Tailwind v4 `source(none)` syntax

### ISSUE THREAD

**Issue by ByteEVM** [biomejs/biome#6441]
### Environment information

```block
CLI:
  Version:                      2.0.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v23.11.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/11.4.1

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true

Workspace:
  Open Documents:               0
```

### What happened?

There's two visible errors while trying to disable automatic source detection in Tailwind v4:
- ✖ expected `,` but instead found `(`
- ✖ Don't use unknown media feature names (`lint/correctness/noUnknownMediaFeatureName`).

The minimal reproduction code is:

```css
@import "tailwindcss" source(none);
```

Docs for [Disabling automatic detection](https://tailwindcss.com/docs/detecting-classes-in-source-files#disabling-automatic-detection).

[Playground link](https://biomejs.dev/playground/?code=QABpAG0AcABvAHIAdAAgACIAdABhAGkAbAB3AGkAbgBkAGMAcwBzACIAIABzAG8AdQByAGMAZQAoAG4AbwBuAGUAKQA7AA%3D%3D&language=css)

### Expected result

It shouldn't error  as it's valid Tailwind v4 syntax.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by siketyan** [comment]
Tailwind uses unofficial CSS syntaxes and we don't support it yet. See #3195 for details.
